tests/unit-tests/test_model_save_load.py

```diff
@@ -201,7 +204,78 @@ def test_sparse_embed_load_corner_cases():
     th.distributed.destroy_process_group()
     dgl.distributed.kvstore.close_kvstore()
 
+class DummySparseEmb:
+    """ Dummy class for sparse embedding
+    """
+    def __init__(
+        self,
+        shape: tuple,
```

Comment:
Use `from typing import Tuple` to retain support for Python 3.8, literal type annotations were introduced in 3.9